Slider/Assets/Scripts/Map/Ocean/NPCRotation.cs

```diff
@@ -0,0 +1,178 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class NPCRotation : MonoBehaviour
+{
+    private List<string> rotationUpdates = new List<string> ();
+    [SerializeField] NPC diceGirl;
+    [SerializeField] NPC diceGuy;
+    [SerializeField] NPC fisherman;
+    [SerializeField] NPC alien;
+    [SerializeField] NPC porker;
+    [SerializeField] NPC traveling_merchant;
+    [SerializeField] NPC catBeard;
+    [SerializeField] NPC fezziwig;
+    [SerializeField] NPC amberOak;
+    [SerializeField] NPC muncher;
+    [SerializeField] Transform tile2Transform;
+    Transform tavernTransform;
+
+    public bool gotBreadge = false;
+    public bool gotCatbeardTreasure = false;
+
+    public Dictionary<string,Vector3> tavernLoc = new Dictionary<string,Vector3>(){
+        {"left_dice", new Vector3(-6, 4, 0)},
+        {"right_dice", new Vector3(-1.5f, 5, 0)},
+        {"left_entrance", new Vector3(-2, -2, 0)},
+        {"right_entrance", new Vector3(4, 2, 0)},
+        {"left_sign", new Vector3(-1.5f, 1.5f, 0)},
+        {"left_left_table", new Vector3(-6, 1.5f, 0)},
+        {"right_sign", new Vector3(3f, 1.5f, 0)},
+        {"outside", new Vector3(20,0,0)},
+        {"coconuts", new Vector3(6.75f, 4f, 0)},
+        {"right_right_table", new Vector3(8.5f, 0.5f, 0)}  
+    };
```

Comment:
it would prob be better if this was set using `Transform`s in inspector somehow but its whatevs

~~also why is "outside" at `(20, 0, 0)`~~ outside should probably be named `out_of_bounds` or sm